features/edit_delete_task.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Edit & Delete Task
+  As a user
+  In order to update my task request
+  I should have the option to edit an open task request
+
+  Background:
+    Given there are two reqesters having several tasks each belonging to some categories
+
+  Scenario: Owner of a task can edit it.
+    Given I am logged in as "requester1@r.com"
+    And I visit the landing page
+    And I click on "View My Tasks"
+    And I click link "Bad computer"
+    And I click on "Edit task"
+    And I fill in field "Task Name" with "Broken computer"
+    And I fill in field "Description" with "faulty powerpack"
+    And I select "Computer" from "Task Category"
+    And I fill in field "Minimum Budget($):" with "500"
+    And I fill in field "Maximum Budget($):" with "700"
+    And I fill in field "Location" with "Gothenbourg"
```

Comment:
Perhaps we should name the fields:
* Minimum Budget:
* Maximum Budget:
And take out the dollar sign, as all countries do not have a dollar currency.